source/run-java-queries.txt

```diff
@@ -0,0 +1,62 @@
+.. _intellij-run-java-queries:
+
```

Comment:
**RE: Your question on linking from Step 6 in the Connect page**

If we choose to link the page, I think it'll be important to provide a little bit more context on what you can do in the Playgrounds generally in relation to MongoDB (i.e. ability to run queries and view results).